lang/python/homeassistant/Makefile

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=homeassistant
+PKG_VERSION:=0.93.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/h/homeassistant
+PKG_HASH:=ecc1e49ff2054194ce4d1b8b7f2d5052943122b8b11c46b5a833e1a66eb46079
+
+PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE.md
+PKG_CPE_ID:=cpe:/a:home-assistant:home-assistant
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/$(PKG_NAME)
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Home Assistant
+  URL:=https://www.home-assistant.io/
+  DEPENDS:= \
+    +python3-aiohttp \
+    +python3-aiohttp-cors \
+    +python3-astral \
+    +python3-async-timeout \
+    +python3-attrs \
+    +python3-bcrypt \
+    +python3-certifi \
+    +python3-cryptography \
+    +python3-ifaddr \
+    +python3-jinja2 \
+    +python3-netdisco \
+    +python3-pip \
+    +python3-pyjwt \
+    +python3-pytz \
+    +python3-pyotp \
+    +python3-slugify \
+    +python3-requests \
+    +python3-ruamel-yaml \
+    +python3-voluptuous \
+    +python3-voluptuous-serialize \
+    +python3-sqlalchemy \
+    +python3-yaml \
+    +python3-zeroconf
```

Comment:
Comparing the list of dependencies here and the ones in [setup.py](https://github.com/home-assistant/home-assistant/blob/0.93.1/setup.py#L34-L53), it looks like you added a few (aiohttp-cors, ifaddr, netdisco, pyotp, sqlalchemy, zeroconf) that are required by [different components](https://github.com/home-assistant/home-assistant/blob/0.93.1/requirements_all.txt) but not core? (Actually I couldn't find any direct requirements for ifaddr.)

If this is the case, I think it would be better to require only core's dependencies and let users decide which components they want to use and then install those component's dependencies individually?